components/derive/src/lib.rs

```diff
@@ -27,32 +27,58 @@ mod tx_set;
 use proc_macro::TokenStream;
 use syn::{Attribute, Lit, Meta, MetaList, MetaNameValue, NestedMeta, Path};
 
-/// Exonum derive attribute names, used as `#[exonum( ATTRIBUTE_NAME = .. )]`
+/// Exonum derive attribute names, used as `#[exonum( ATTRIBUTE_NAME = .. or ATTRIBUTE_NAME )]`
 const CRATE_PATH_ATTRIBUTE: &str = "crate";
 const PB_CONVERT_ATTRIBUTE: &str = "pb";
+const SERDE_PB_CONVERT_ATTRIBUTE: &str = "serde_pb_convert";
 
 /// Derives `ProtobufConvert` trait.
+///
 /// Attributes:
-/// `#[exonum( pb = "path" )]`
-/// Required. `path` is name of the corresponding protobuf struct(generated from .proto file)
-/// `#[exonum( crate = "path" )]`
-/// Optional. `path` is prefix of the exonum crate(usually "crate" or "exonum")
+///
+/// * `#[exonum( pb = "path" )]`
+/// Required. `path` is the name of the corresponding protobuf generated struct.
+///
+/// * `#[exonum( crate = "path" )]`
+/// Optional. `path` is prefix of the `exonum` crate(usually "crate" or "exonum").
+///
+/// * `#[exonum( serde_pb_convert )]`
+/// Optional. Implements `serde::{Serialize, Deserialize}` using structs that were generated with
+/// protobuf.
+/// ```json
+/// // For example struct with `exonum::crypto::Hash` with this
```

Comment:
Nit: there are no comments in JSON, so this may render weirdly in the docs.

Reply:
I think it could be supported in the future, but I found issue about this closed.
I'll replace it with `text`